infra/lib/fhir-server-stack.ts

```diff
@@ -65,7 +65,7 @@ export function settings(): Settings {
     return {
       ...defaults,
       cpu: 1 * vCPU,
-      memoryLimitMiB: 1024,
+      memoryLimitMiB: 2048,
```

Comment:
1 vCPU requires at least 2GB of RAM - https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_size